data/mupen64plus.ini

```diff
@@ -13107,6 +13107,14 @@ GoodName=Star Wars - Rogue Squadron (U) (M3) [t1]
 CRC=165CBDA2 05A62246
 RefMD5=47CAC4E2A6309458342F21A9018FFBF0
 
+[2E458D7CC355D7918493B0E0362C9A20]
+GoodName=Star Wars - Rogue Squadron (U) (M3) (V1.1) [!]
+CRC=C7F30CFA ECB0FA36
+Players=1
```

Comment:
@AmbientMalice in case that didn't make sense (I know the INI file works different than PJ64), you can just remove:
```
Players=1
SaveType=Eeprom 4KB
Rumble=Yes
CountPerOp=1
```
and add 
```
RefMD5=47CAC4E2A6309458342F21A9018FFBF0
```
and it will copy the settings from the current U entry for Rogue Squadron